src/BinSkim.Rules/PERules/BA2014.DoNotDisableStackProtectionForFunctions.cs

```diff
@@ -59,6 +59,9 @@ private static StringSet BuildApprovedFunctionsStringSet()
                 "_TlgWrite",
                 "__vcrt_trace_logging_provider::_TlgWrite"
             };
+
+            // Also allow Gs initialization functions.
+            result.UnionWith(StackProtectionUtilities.GSInitializationFunctionNames);
```

Comment:
>result.UnionWith(StackProtectionUtilities.GSInitializationFunctionNames); [](http://example.com/codeflow?start=12&length=73)

Is there any way we could get unit test(s) to cover this change?

Reply:
added